README.md

```diff
@@ -152,140 +201,265 @@ If your server is configured with [authentication enabled](https://openfga.dev/d
 ```
 
-#### Delete Store
+##### Delete Store
+
+Delete a store.
 
 [API Documentation](https://openfga.dev/api/service/docs/api#/Stores/DeleteStore)
 
 > Requires a client initialized with a storeId
 
 ```csharp
-var store = await openFgaApi.DeleteStore();
+var store = await fgaClient.DeleteStore();
 ```
 
-#### Write Authorization Model
+#### Authorization Models
+
+##### Read Authorization Models
+
+Read all versions of the authorization model.
+
+[API Documentation](https://openfga.dev/api/service#/Authorization%20Models/ReadAuthorizationModels)
+
+```csharp
+var options = new ClientReadAuthorizationModelsOptions {
+    PageSize = 10,
+    ContinuationToken = "...",
+};
+var response = await fgaClient.ReadAuthorizationModels(options);
+
+// response.AuthorizationModels = [
+// { id: "1uHxCSuTP0VKPYSnkq1pbb1jeZw", schema_version: "1.1", type_definitions: [...] },
+// { id: "GtQpMohWezFmIbyXxVEocOCxxgq", schema_version: "1.1", type_definitions: [...] }];
+```
+
+##### Write Authorization Model
+
+Create a new version of the authorization model.
 
 [API Documentation](https://openfga.dev/api/service#/Authorization%20Models/WriteAuthorizationModel)
 
 > Note: To learn how to build your authorization model, check the Docs at https://openfga.dev/docs.
 
 > Learn more about [the OpenFGA configuration language](https://openfga.dev/docs/configuration-language).
 
+> You can use the [OpenFGA Syntax Transformer](https://github.com/openfga/syntax-transformer) to convert between the friendly DSL and the JSON authorization model.
+
 ```csharp
-var documentRelations = new Dictionary<string, Userset>()
-{
-    {"writer", new Userset(_this: new object())},
-    {
-        "viewer",
-        new Userset(union: new Usersets(new List<Userset>()
-            {new(new object(), new ObjectRelation("", "writer"))}))
+var body = new WriteAuthorizationModelRequest {
+    SchemaVersion = "1.1",
+    TypeDefinitions = new List<TypeDefinition>() {
+        new() {
+            Type = "user",
+            Relations = new Dictionary<string, Userset>() {}
+        },
+        new() {
+            Type = "document",
+            Relations = new Dictionary<string, Userset>()
+                {
+                    {"writer", new Userset(_this: new object())},
+                    {
+                        "viewer",
+                        new Userset(union: new Usersets(new List<Userset>()
+                            {new(new object(), new ObjectRelation("", "writer"))}))
+                    }
+                },
+            Metadata = new Metadata() {
+                Relations = new Dictionary<string, RelationMetadata>() { {
+                    "viewer", new RelationMetadata() { DirectlyRelatedUserTypes = new List<RelationReference> {
+                        new () { Type = "user"}
+                    } }
+                } }
+            },
+        }
```

Comment:
This model gives `(line 1:314): error parsing &#34;computedUserset&#34;, oneof openfga.v1.Userset.userset is already set`